src/content/docs/backend/API/unit-group_sets.md

```diff
@@ -0,0 +1,360 @@
+  | group_id     | id of the group      | path           | integer   | Yes       |
+  | id           | id of the member     | path           | integer   | Yes       |
+
+- Response: `200 OK`
+
+- Example Request:
+
+  ```bash
+   curl -X DELETE --header 'Accept: application/json' --header 'Username: aadmin' --header 'Auth_Token: Say13yA497H8wSFFXZxh' 'http://localhost:3000/api/units/1/group_sets/2/groups/3/members/4'
+- Response Body:
+  ```
+        True
+  ```
+---
+
+### POST : Add a member to a group
+
+POST /api/units/{unit_id}/group_sets/{group_set_id}/groups/{group_id}/members/{project_id}
+
+- URL: `/api/units/{unit_id}/group_sets/{group_set_id}/groups/{group_id}/members/{project_id}`
+- Method: `POST`
+- Parameter:
+
+| Parameter    | Description              | Parameter Type | Data Type | Mandatory |
+| ------------ | ------------------------ | -------------- | --------- | --------- |
+| Username     | Username                 | header         | string    | Yes       |
+| Auth_Token   | Authentication token     | header         | string    | Yes       |
+| unit_id      | The unit id              | path           | integer   | Yes       |
+| group_set_id | Id of the group set      | path           | integer   | Yes       |
+| group_id     | id of the group          | path           | integer   | Yes       |
+| project_id   | project id of the member | path           | integer   | Yes       |
+
+- Response: `200 OK`
+
+- Example Request:
+  ```bash
+   curl -X POST --header 'Accept: application/json' --header 'Username: aadmin' --header 'Auth_Token: Say13yA497H8wSFFXZxh' 'http://localhost:3000/api/units/1/group_sets/2/groups/3/members/4'
+  ```
+- Response Body:
+
+  ```json
+  {
+    "student_username": "TESTREQ",
+    "id": 4,
+    "student_first_name": "TEST",
+    "student_last_name": "REQ",
+    "student_nickname": "TEST1",
+    "target_grade": "A"
+  }
+  ```
+---
+### GET: Get the members of the group
+
+GET /api/units/{unit_id}/group_sets/{group_set_id}/groups/{group_id}/members
+
+- URL: `/api/units/{unit_id}/group_sets/{group_set_id}/groups/{group_id}/members`
+- Method: `GET`
+- Parameter:
+
+| Parameter    | Description          | Parameter Type | Data Type | Mandatory |
+| ------------ | -------------------- | -------------- | --------- | --------- |
+| Username     | Username             | header         | string    | Yes       |
+| Auth_Token   | Authentication token | header         | string    | Yes       |
+| unit_id      | The unit id          | path           | integer   | Yes       |
+| group_set_id | Id of the group set  | path           | integer   | Yes       |
+| group_id     | id of the group      | path           | integer   | Yes       |
+
+- Response: `200 OK`
+
+- Example Request:
+  ```bash
+   curl -X GET --header 'Accept: application/json' --header 'Username: aadmin' --header 'Auth_Token: Say13yA497H8wSFFXZxh' 'http://localhost:3000/api/units/1/group_sets/2/groups/3/members'
+  ```
+- Response Body:
```

Comment:
The return object for this endpoint specifies that it only presents :student :id and :target_grade, :student may include username, first_name, last_name, nickname, i just can't find it in the code

Reply:
Hi Jake, I have now changed the response body with just Student, Id and target grade as per the code.